python/controls/window-drag-area/no-frame-window.py

```diff
@@ -19,10 +19,11 @@ def main(page: Page):
                     ),
                     expand=True,
                 ),
-                IconButton(icons.CLOSE, on_click=lambda _: page.window_close()),
+                # Use the updated method to close the window
+                IconButton(icons.CLOSE, on_click=lambda _: page.window.close()),
```

Comment:
```suggestion
                IconButton(icons.CLOSE, on_click=lambda e: page.window.close()),
```

Let's have `e` there just for good practice.